nautilus_trader/adapters/dydx/http/errors.py

```diff
@@ -54,7 +55,7 @@ def should_retry(error: BaseException) -> bool:
 
     """
     if isinstance(error, DYDXGRPCError):
-        return True
+        return error.code in DYDX_RETRY_ERRORS_GRPC
```

Comment:
Are there rate limit or network related error codes which we'd want to retry on?
I'll have a look at the dYdX API myself when I can get to it.

Reply:
Yes there are rate limits. This list can be expanded for sure. As of now, we make a distinction between RPC errors, HTTP errors and also catch exceptions thrown by the grpcio library (unavailable errors). 

HTTP requests not use the retry manager yet.